ui/shared/HelpMenu.jsx

```diff
@@ -80,6 +80,7 @@ const HelpMenu = () => (
           target="_blank"
           rel="noopener noreferrer"
           href={item.href}
+          key={item.text}
```

Comment:
This was throwing those "need a uniquey key errors" messages.  So I fixed it here.